sparkles/tests/test_review.py

```diff
@@ -305,25 +306,29 @@ def test_roll_options_dec89_9():
 
     # Expected roll options.  Note same basic outputs for add_ids and drop_ids but
     # difference roll values.
+
+    # NOTE ALSO: the P2 values are impacted by the bad region aroundrow=-318,
+    # col=-298. If handling of that bad region for acq changes then the P2
+    # values may change.
     exp = {}
     exp[48000] = [' roll   P2  n_stars improvement roll_min roll_max  add_ids   drop_ids',
                   '------ ---- ------- ----------- -------- -------- --------- ---------',
                   '287.25 3.61    0.55        0.00   287.25   287.25        --        --',
-                  '281.00 7.44    6.98        9.64   276.75   285.25 608567744        --',
+                  '281.00 7.24    6.98        9.53   276.75   285.25 608567744        --',
                   '287.50 7.25    5.43        7.68   268.50   306.00        --        --',
                   '268.50 6.82    4.98        6.93   268.50   273.25 610927224 606601776',
                   '270.62 6.82    4.22        6.01   268.50   273.25 610927224        --']
 
     exp[18000] = [' roll   P2  n_stars improvement roll_min roll_max  add_ids   drop_ids',
                   '------ ---- ------- ----------- -------- -------- --------- ---------',
                   '276.94 3.61    7.54        0.00   276.94   276.94        --        --',
-                  '270.57 7.44    8.00        1.99   266.19   274.94 608567744        --',
                   '277.07 7.25    8.00        1.89   258.19   295.69        --        --',
+                  '270.57 7.16    8.00        1.84   266.19   274.94 608567744        --',
                   '258.19 6.82    8.00        1.68   258.19   262.69 610927224 606601776',
                   '259.69 6.82    8.00        1.68   258.19   262.69 610927224        --']
 
     for obsid in (48000, 18000):
-        kwargs = mod_std_info(att=att, n_guide=8, obsid=obsid, date=date)
+        kwargs = mod_std_info(att=att, n_guide=8, n_fid=0, obsid=obsid, date=date)
```

Comment:
Explicitly specify zero fids.

Reply:
It wouldn't matter to specify n_fid=3 and n_guide=5 for the OR obsid but this seems simpler if the real point of the two types is to get different roll choices based on the default science target offsets.  For 48000 I wasn't sure if detector should also be specified to be more like every other ER?  But if you change the n_fid default in proseco I guess it doesn't matter.  I suppose I just found it interesting that proseco and sparkles have different strategies about deciding if something is/was an OR.

Reply:
shouldn't this be an internal constraint?

Just wanted to add some noise 😄

Reply:
By internal constraint, you mean that n_fid + n_guide > 8 should throw an error at the start?  I think Tom's fix in proseco to unset the n_fid default actually does have that effect in most cases.

Reply:
> I suppose I just found it interesting that proseco and sparkles have different strategies about deciding if something is/was an OR.

Basically proseco does not have the concept of OR/ER at all. There is only one place that pays attention to OR/ER: `get_monitors` which gets monitor window commanding for an existing starcheck catalog text.